Makefile.d/test.mk

```diff
@@ -105,6 +105,7 @@ gotests/patch: \
 	@$(call green, "apply patches to go test files...")
 	find $(ROOTDIR)/internal/k8s/* -name '*_test.go' | xargs sed -i -E "s%k8s.io/apimachinery/pkg/api/errors%github.com/vdaas/vald/internal/errors%g"
 	find $(ROOTDIR)/* -name '*_test.go' | xargs sed -i -E "s%cockroachdb/errors%vdaas/vald/internal/errors%g"
```

Comment:
can we remove this line?

Reply:
I think so but if we leave this code, there is no problem for vald's code quality.
I think it's better to leave this code and remove it as a the feature work